src/PackageAnalyzer.jl

```diff
@@ -183,17 +183,20 @@ function Base.show(io::IO, p::PackageV1)
         """
         if !isempty(p.lines_of_code)
             l_src = sum_julia_loc(p, "src")
+            l_ext = sum_julia_loc(p, "ext")
             l_test = sum_julia_loc(p, "test")
             l_docs = sum_doc_lines(p)
             l_readme = sum_readme_lines(p)
 
-            p_test = @sprintf("%.1f", 100 * l_test / (l_test + l_src))
-            p_docs = @sprintf("%.1f", 100 * l_docs / (l_docs + l_src))
+            p_ext = @sprintf("%.1f", 100 * l_ext / (l_test + l_src + l_ext))
+            p_test = @sprintf("%.1f", 100 * l_test / (l_test + l_src + l_ext))
+            p_docs = @sprintf("%.1f", 100 * l_docs / (l_docs + l_src + l_ext))
```

Comment:
Maybe we should have
```suggestion
            l_total = l_test + l_src + l_ext
            p_ext = @sprintf("%.1f", 100 * l_ext / l_total)
            p_test = @sprintf("%.1f", 100 * l_test / l_total)
            p_docs = @sprintf("%.1f", 100 * l_docs / l_total)
```
so that we don't have to change the total in multiple places every time.

Reply:
Is it okay to have `p_docs` is larger than 100? Note that not all denominators are equal. Should we define `l_total` with
```julia
l_total = l_test + l_docs + l_src + l_ext
```
?

Reply:
Ah, yes, sorry, I missed that one was different